docs/support/faq.rst

```diff
@@ -10,17 +10,21 @@ Yes, absolutely! Use Streetmix however you like, but :ref:`please provide attrib
 What is the end user license?
 -----------------------------
 
-Streetmix has adopted two licenses: the **software** uses the `BSD license`_, while the **content** uses a `Creative Commons license`_. One or both of these licenses might apply to your situation. These licenses give you some essential rights for using Streetmix without having to explicitly ask us for permission.
+Streetmix has adopted two licenses: the **software** uses the `GNU Affero General Public License v3.0`_ (AGPL), while the **content** uses a `Creative Commons license`_. One or both of these licenses might apply to your situation. These licenses give you some essential rights for using Streetmix without having to explicitly ask us for permission.
```

Comment:
Useful to say *which* Creative Commons license, as they are very different.  CC-BY and CC-BY-SA are the two that are "free as in free software / open as in open source".  By contrast, CC-BY-NC, CC-BY-ND, and CC-BY-NC-ND are non-free.  (And CC-0 is just a special-purpose outlier that you shouldn't use unless you really need it.)

Reply:
Ah, good call. The CC license we use is CC-BY-SA, and we specify it elsewhere but it's a good idea to clarify this here too. Thanks!